query-engine/core/src/query_ast/read.rs

```diff
@@ -96,6 +104,7 @@ pub struct RecordQuery {
     pub nested: Vec<ReadQuery>,
     pub selection_order: Vec<String>,
     pub aggregation_selections: Vec<RelAggregationSelection>,
+    pub options: BitFlags<QueryOption>,
```

Comment:
It's so much elegant and better now, @Weakky . Thank you so much for the suggestion!